crates/providers/lattice-controller/.gitignore

```diff
@@ -1,26 +1,16 @@
-# This file lists build byproducts, 
+# This file lists build byproducts,
 # IDE-specific files (unless shared by your team)
 
-#
-
 ## Build
-/build
-/dist/
 /target
 **target
 
-## File system
-.DS_Store
-desktop.ini
-
 ## Editor
 *.swp
 *.swo
 Session.vim
 .cproject
-.idea
```

Comment:
I would keep this file around as a convenience to anyone working on this project in the community.

Reply:
Done! will re-add

- [ ] re-add `.idea` to `lattice-controller/.gitignore`